pom.xml

```diff
@@ -233,24 +253,9 @@
                 <version>2.1.0-374</version>
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi.common</groupId>
-                <artifactId>util-moduleloader</artifactId>
-                <version>4.0.0-SNAPSHOT</version>
-            </dependency>
-            <dependency>
-                <groupId>com.github.cafapi.common</groupId>
-                <artifactId>util-naming</artifactId>
-                <version>4.0.0-SNAPSHOT</version>
-            </dependency>
-            <dependency>
-                <groupId>com.github.cafapi.common</groupId>
+                <groupId>com.github.cafapi.util</groupId>
                 <artifactId>util-process-identifier</artifactId>
-                <version>4.0.0-SNAPSHOT</version>
-            </dependency>
-            <dependency>
-                <groupId>com.github.cafapi.common</groupId>
-                <artifactId>util-ref</artifactId>
-                <version>4.0.0-SNAPSHOT</version>
+                <version>3.2.0-684</version>
```

Comment:
@raghavendran-gopalakrishnan This is interesting.  The dependency tooling is removing `com.github.cafapi.common:util-process-identifier:4.0.0-SNAPSHOT` and re-adding `com.github.cafapi.util:util-process-identifier:3.2.0-684`!